test/blackbox-tests/test-cases/dune-init.t/run.t

```diff
@@ -358,3 +532,8 @@ Can init and build a project using Esy
   test_esy_proj/test:
   dune
   test_esy_proj.ml
+
+And the dune-project file does NOT specify generation of an opam file:
+
+  $ cat test_esy_proj/dune-project | grep "generate_opam_files"
+  (generate_opam_files false)
```

Comment:
Not sure what other properties should hold of dune-project files in the case of esy projects.